src/flint/fmpz_mat.jl

```diff
@@ -1894,15 +1887,17 @@ function mul!(z::ZZMatrix, x::ZZMatrix, y::ZZMatrix, fl::Bool = false)
 end
 
 function mul!(y::ZZMatrix, x::Int)
+  mul!(y, y, x)
```

Comment:
As stated in https://github.com/Nemocas/AbstractAlgebra.jl/issues/1736 I think we should just define once and for all something like this in AA:
```
add!(a,b) = add!(a,a,b)
sub!(a,b) = sub!(a,a,b)
mul!(a,b) = mul!(a,a,b)
div!(a,b) = div!(a,a,b)
rem!(a,b) = rem!(a,a,b)
mod!(a,b) = mod!(a,a,b)
neg!(a) = neg!(a,a)
inv!(a) = inv!(a,a)
```
and probably a bunch more